runbot_website_display/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================
+Runbot display options
+======================
+
+This module was written to allow users to reorder runbot branches and to hide 
+uninteresting ones.
+
+Configuration
+=============
+
+After installation, a runbot repository has a checkbox `website_published` you
+can turn off if you don't want to see a repository on the website.
```

Comment:
After installation, a runbot repository has a checkbox `website_published` that you
can turn off if you don't want to see the repository on the website.